docs/cloud/cloud-org/cloud-org-manage.md

```diff
@@ -21,16 +22,20 @@ has_toc: false
 
 ## How do I change my organization name and address?
 
-* [Learn how to change organization details](/docs/cloud/cloud-org/cloud-org-address)
+- [Learn how to change organization details](/docs/cloud/cloud-org/cloud-org-address)
 
 ## How do I change my billing address for my account?
 
 {% include /cloud-org/cloud-summary-billing-contact.md %}
 
-* [Learn how to update billing details](/docs/cloud/cloud-org/cloud-org-update-billing)
+- [Learn how to update billing details](/docs/cloud/cloud-org/cloud-org-update-billing)
 
 ## How do I change my technical contact details?
 
 {% include /cloud-org/cloud-summary-tech-contact.md %}
 
-* [Learn how to update technical contact address](/docs/cloud/cloud-org/cloud-org-update-tech-contact)
+- [Learn how to update technical contact address](/docs/cloud/cloud-org/cloud-org-update-tech-contact)
+
+## How do I upgrade my organization to paid?
```

Comment:
Again here -- a paid **WHAT**??

We're talking about a paid **account**, so say that.